stable/meteor/templates/cronjob.yaml

```
@@ -17,140 +17,147 @@ spec:
       template:
         metadata:
           labels:
-{{- if .Values.labels }}
-{{ toYaml .Values.labels | indent 12 }}
-{{- end }}
+            {{- if .Values.labels }}
+            {{ toYaml .Values.labels | indent 12 }}
+            {{- end }}
         spec:
           {{- if .Values.serviceAccount.name }}
           serviceAccountName: {{ .Values.serviceAccount.name }}
           {{- end }}
           containers:
-          - name: "{{ include "meteor.name" . }}"
-            image: "{{ required `image.repository is required` .Values.image.repository }}:{{ .Values.image.tag }}"
-            imagePullPolicy: {{ .Values.image.pullPolicy }}
-            command:
-              - /bin/sh
-              - -c
-              - |
-                meteor run /opt/recipes;
-            {{- if .Values.telegraf.enabled }}
-                pkill -SIGTERM telegraf
-            {{- end }}
-            {{- if .Values.otelcollector.enabled }}
-                pkill -SIGTERM otelcol-contrib
-            {{- end }}
-            volumeMounts:
-              - name: "{{ include "meteor.name" . }}-volume"
-                mountPath: /opt/recipes
-              {{- range $k, $v := .Values.ssl_secrets }}
-              - name: {{ $v }}
-                mountPath: {{ printf "%s/%s" "/etc/secret" $v }}
-              {{- end }}
-              {{- if .Values.serviceAccount.tokenProjection.enabled }}
-              - name: {{ .Values.serviceAccount.name }}
-                mountPath: {{ .Values.serviceAccount.tokenProjection.mountPath }}
-              {{- end}}
-            envFrom:
-              - configMapRef:
-                  name: "{{ include "meteor.name" . }}-variables-configmap"
-            {{- if .Values.secretConfig }}
-              - secretRef:
-                  name: "{{ include "meteor.name" . }}-secret"
-            {{- end }}
-            resources:
-              {{- toYaml .Values.resources | nindent 14 }}
+            - name: "{{ include "meteor.name" . }}"
+              image: "{{ required `image.repository is required` .Values.image.repository }}:{{ .Values.image.tag }}"
+              imagePullPolicy: {{ .Values.image.pullPolicy }}
+              command:
+                - /bin/sh
+                - -c
+                - |
+                  meteor run /opt/recipes;
+                  METEOR_EXIT_CODE=$?
+
+                  if [ $METEOR_EXIT_CODE -eq 0 ]; then
+                    {{- if .Values.postCmd.enabled }}
+                    {{ .Values.postCmd.cmd | indent 20 }}
+                    {{- end }}
+                  fi
```

let's move if higher
```suggestion
                  {{- if .Values.postCmd.enabled }}
                  METEOR_EXIT_CODE=$?

                  if [ $METEOR_EXIT_CODE -eq 0 ]; then
                    {{ .Values.postCmd.cmd | indent 20 }}
                  fi
                  {{- end }}
```